serokell-util.cabal

```diff
@@ -97,6 +97,7 @@ library
                        RecordWildCards
                        DeriveDataTypeable
                        GeneralizedNewtypeDeriving
+                       NoImplicitPrelude
```

Comment:
I think you should specify `^>= 1.0.4` version for `universum` in dependencies. And add latest `universum` in `extra-deps`. `serokell-util` is not on Stackage so no problems here.